webapp/src/main/webapp/src/app/sender/archive/sender-archive.component.html

```diff
@@ -0,0 +1,74 @@
+<!--View sync message overall status-->
+<div>
+	<div class="row">
+		<div class="col-md-8">
+		</div>
+		<div class="col-md-4">
+			<div>
+				<table style="left: 0;">
+					<tr>
+						<td></td>
+						<td colspan="4"></td>
+						<td colspan="2" i18n="@@common-start">Start Date</td>
+						<td></td>
+						<td i18n="@@common-end">End Date </td>
+						<td> </td>
+					</tr>
+					<tr>
+						<td >
+							<button class="btn btn-outline-primary" (click)="searchByPeriod($event)">
+								<span i18n="@@common-search-period">Search by Period</span>
```

Comment:
I think these should Explicitly say Search By Event Date and renamed all component, service and back end controller method accordingly, this search is specific to sender sync archives